zone/questmgr.h

```diff
@@ -34,27 +34,30 @@ namespace EQ
 }
 
 class QuestManager {
-	struct running_quest {
-		Mob *owner;
-		Client *initiator;
-		EQ::ItemInstance* questitem;
-		const SPDat_Spell_Struct* questspell;
-		bool depop_npc;
-		std::string encounter;
-	};
 
 	struct PausedTimer {
 		Mob*        owner;
 		std::string name;
 		uint32      time;
 	};
 public:
+
+	struct RunningQuest {
```

Comment:
Take this out of the greater class if it's going to be used outside of the greater class.